Assets/Scripts/Entities/Jackie.cs

```diff
@@ -107,8 +117,10 @@ override public void HandleUseCard(ActionClass a)
         discard.Add(used);
         used.transform.position = new Vector3(500, 500, 1);
         // draw a replacement card
-        RenderHand();
-    }
+        // RenderHand(); 
+        HighlightManager.RenderHandIfAppropriate(this);
```

Comment:
nit, but im not a fan of how RenderHandIfAppropriate is called so often. I would rather have one instance of RenderHandIfAppropriate inside RenderHand() that returns a boolean whether the player can render a hand or not.
Then if the boolean is false, RenderHand() just returns early

Reply:
maybe I'm misunderstanding but I did this because Highlight manager keeps track of the current player. If this makes more sense to me I'll def refactor